### PR TITLE
Update (RMB 33.2.7, etc.)

### PR DESCRIPTION
**Pull request by julianladisch**

### CODE REVIEW

**Review by steveellis — APPROVED** [review]

File: src/main/resources/archetype-resources/Jenkinsfile [hunk 1]
```diff
@@ -1,14 +1,20 @@
 buildMvn {
-  publishModDescriptor = 'no'
-  runLintRamlCop = 'no'
-  publishAPI = 'no'
-  mvnDeploy = 'no'
+  publishModDescriptor = false
+  mvnDeploy = false
+  publishAPI = false // Deprecated. See doApiDoc.
+  runLintRamlCop = false // Deprecated. See doApiLint.
```
Comment (dcrossleyau) [inline src/main/resources/archetype-resources/Jenkinsfile:5]: I reckon just remove these two deprecated properties. It still seems to confuse some people.

(I will continue adjustment of the dev.f.o docs, which attempt to get projects to transition to the new tools.)

**Review by dcrossleyau — CHANGES_REQUESTED** [review]
Thanks Julian. It did need some attention.

Please also adjust the date range of the main README, and adjust the archetype README to be 2022.

File: src/main/resources/archetype-resources/Jenkinsfile (hunk 1 above)
Comment (julianladisch) [reply src/main/resources/archetype-resources/Jenkinsfile]: Thanks, done!

**Comment by julianladisch** [comment]
Years haven been bumped.

**Review by adamdickmeiss — APPROVED** [review]

**Review by dcrossleyau — APPROVED** [review]

**Pull request merged**